examples/calibrate_your_rule.py

```diff
@@ -27,19 +27,12 @@
 from sklearn.pipeline import make_pipeline
 
 
-def my_first_infertrade_rule(df: pd.DataFrame) -> pd.DataFrame:
-    """Simple example rule."""
-    df["allocation"] = 0.0
-    df["allocation"][df["price"].pct_change() > 0.05] = 0.5
-    return df
-
```

Comment:
I'm not necessarily in favor of delete this function, but I noticed that it is present in both example scripts and is used in neither.